lua/lib/opensn.cc

```diff
@@ -160,29 +160,8 @@ static bool reg = opensnlua::Console::Bind(
       .beginClass<std::shared_ptr<Vector3>>("Vector3Ptr")
       .endClass();
 
-    luabridge::getGlobalNamespace(L)
-      .beginNamespace("squad")
-      .beginClass<GaussQuadrature>("GaussQuadrature")
-      .addProperty("qpoints", &GaussQuadrature::qpoints)
-      .addProperty("weights", &GaussQuadrature::weights)
-      .endClass()
-      .deriveClass<GaussLegendreQuadrature, GaussQuadrature>("GaussLegendreQuadrature")
-      .addStaticFunction("Create", &GaussLegendreQuadrature::Create)
-      .endClass()
-      .beginClass<std::shared_ptr<GaussLegendreQuadrature>>("GaussLegendreQuadraturePtr")
-      .endClass()
-      .deriveClass<GaussChebyshevQuadrature, GaussQuadrature>("GaussChebyshevQuadrature")
-      .addStaticFunction("Create", &GaussChebyshevQuadrature::Create)
-      .endClass()
-      .beginClass<std::shared_ptr<GaussChebyshevQuadrature>>("GaussChebyshevQuadraturePtr")
-      .endClass()
-      .endNamespace();
-
     luabridge::getGlobalNamespace(L)
       .beginNamespace("aquad")
-      .addFunction("Legendre", &Legendre)
-      .addFunction("LegendreDerivative", &dLegendredx)
-      .addFunction("Ylm", &Ylm)
```

Comment:
I am sorry, I didn't reply to your slack message. I think we need to keep user lua access to Legendre and Ylm. I will be using this soon in a Tutorial with class.

Reply:
`Legendre` and `Ylm` are back.